### ISSUE TITLE
Presentation flickering when viewed on Google Chrome.

### ISSUE THREAD

**Issue by simolev** [jitsi/jitsi-meet#7274]
### Description:

When screen or app is shared by a user, the other user sees it with a continuous, very annoying flickering. It looks as if the image were switching rapidly between low and high resolution.

This behaviour has been noticed by several users in our organization in one-on-one calls, when the presenter browser is Google Chrome or Mozilla Firefox and the viewer browser is Google Chrome. It does not happen when the viewer browser is Mozilla Firefox.

### Steps to reproduce:

1. Presenter clicks on the Share Screen button, selects what they want to share, and start sharing.
2. Participant on Google Chrome sees the presentation but gets a headache from the flickering.
This happens consistently: every time the feature is used and the viewer is on Google Chrome.

### Expected behavior:

The viewer should receive a decent representation of the presenter's desktop or app.

### Actual behavior:

The viewer sees what they are supposed to see, but the image is flickering heavily, rendering the feature unusable.

![screen-sharing](https://user-images.githubusercontent.com/13628396/87026293-d3955800-c1db-11ea-999b-4cc3d081e47a.gif)

![screen-sharing](https://user-images.githubusercontent.com/13628396/87027457-7bf7ec00-c1dd-11ea-9a90-d4392436cb0d.gif)

### Server information:

- Jitsi Meet version: jitsi-meet/stable,now 2.0.4627-1 from the Ubuntu/Debian repository.
- Operating System: Ubuntu 20.04 LTS.

Also observed on the public installations at https://meet.jit.si and https://meet.sangoma.com

### Client information:

- Browser / app version: Google Chrome 83.0.4103.116 64 bit
- Operating System: Windows 10

**Comment by saghul** [comment]
Can you test it here: https://webrtc.github.io/samples/src/content/getusermedia/getdisplaymedia/ and see if you also get the flickering? If you do, then it must be a browser issue and nothing we can act upon.

**Comment by simolev** [comment]
@saghul The recording is flawless, I just don't know whether I should also see the shared screen in real time instead of the black box.

**Comment by simolev** [comment]
Today our (we are still test-driving them) virtual PBX provider moved us to their new platform: their video-conferencing solution is now based on Jitsi Meet! Even more thrilling, there is no flickering when viewing a presented window on Google Chrome, **and** Firefox can finally terminate screen sharing by pressing the presentation button again ([6991](https://github.com/jitsi/jitsi-meet/issues/6991)) . Any clue on how they might have solved these two issue?

**Comment by saghul** [comment]
We've done a number of improvements to screen-sharing over time. I suppose they were running an old version of the code and have updated now.

**Comment by simolev** [comment]
I'm glad there have been improvements over time :-)

* If you are saying that I'm using old code from the Ubuntu/Debian repository, please point me to instructions on how to update it (I believe I'm using the latest version available in the repository).
* Why would https://meet.jit.si use old code?

**Comment by simolev** [comment]
Can you please reopen this issue until it has been fixed?
Having seen an installation where the issue is absent does not automatically mean that it is solved.
I'm still experiencing the problem on my installation and on https://meet.jit.si.

**Comment by simolev** [comment]
Updated to jitsi-meet/stable,now 2.0.4857-1 but the issue persists.

**Comment by chris-est** [comment]
I have the same flickering when using my own server (v2.0.5142-1 on debian) or the official instance https://meet.jit.si.
Tested on Windows and Linux with Firefox, Chromium and the Jitsi Electron App.

As mentioned i also run my own server so i am willing to test different settings if there are any that can help improve this.

> Can you test it here: https://webrtc.github.io/samples/src/content/getusermedia/getdisplaymedia/

I have tested this and there is no flickering when running this test.
